### DISCUSSION TITLE
SevenGUIs update

### DISCUSSION THREAD

**Discussion by zaceno** [jorgebucaran/hyperapp#1008] [Show and tell]
Posted this in the slack but it didn't seem to get any attention so I'm trying here. It's not a big deal or anything – just looking for some review 🙏 

I updated my seven guis implementation to improve the presentation (including code examples), improve some of the implementation. Oh and actually make them work with the newest hyperapp 😉 .

So yeah, I'd really like your feedback on how the implementations could be improved primarily, but feedback on the presentation and how the overall project is structured is welcome too. 

https://zaceno.github.io/sevenguis-hyperapp/

Also anyone is free to chime in – not just seasoned hyperappers! In fact, I'd say the main target audience for this is the _un_ seasoned hyperapper.

**Comment by zaceno** [comment]
Beacuse of #1000 I am currently using a patched version of hyperapp with a userland stoppable-HOA. If we got #873 merged, or something equivalent, I could use straight-up regular hyperapp btw. Just saying 😉

**Reply by jorgebucaran** [reply]
Should be noted that this is now just using Hyperapp as #873 was resolved. 🙌

**Comment by Seanmcdon** [comment]
I think this is great. I look forward to studying it.  Having the code along side is great. I'm not a full time developer but would like to get into it. This is helpful
